util/session/sessionmanager.go

```diff
@@ -84,6 +85,9 @@ func (mgr *SessionManager) Create(subject string, secondsBeforeExpiry int64, id
 	// Create a new token object, specifying signing method and the claims
 	// you would like it to contain.
 	now := time.Now().UTC()
+	if id == "" {
+		id = strconv.FormatInt(now.Unix(), 10)
```

Comment:
Please use UUID instead. This is consistent with account tokens and also unix time stamp is not best unique identifier.

https://github.com/argoproj/argo-cd/blob/783399b3f677b0aca90f9f41d8eeef6a286ba72c/server/account/account.go#L190